patches/chrome-browser-resources-settings-clear_browsing_data_dialog-clear_browsing_data_dialog.html.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html
+index 1a086904cc3c93cb0ff6e0ad0eab0de850638591..55399a8db294850eb008a1a23caade99bb1dde7f 100644
+--- a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html
++++ b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html
+@@ -17,6 +17,7 @@
+ <link rel="import" href="../route.html">
+ <link rel="import" href="../settings_shared_css.html">
+ <link rel="import" href="../settings_vars_css.html">
++<link rel="import" href="../brave_clear_browsing_data_on_exit_page/brave_clear_browsing_data_on_exit_page.html">
+ 
+ <dom-module id="settings-clear-browsing-data-dialog">
+   <template>
+@@ -117,6 +118,7 @@
+             selected="{{prefs.browser.last_clear_browsing_data_tab.value}}">
+           <paper-tab id="basicTabTitle">$i18n{basicPageTitle}</paper-tab>
+           <paper-tab id="advancedTabTitle">$i18n{advancedPageTitle}</paper-tab>
++          <paper-tab id="onExitTabTitle">$i18n{onExitPageTitle}</paper-tab>
+         </paper-tabs>
+       </div>
+       <div slot="body">
+@@ -242,6 +244,11 @@
+                 disabled="[[clearingInProgress_]]">
+             </settings-checkbox>
+           </div>
++          <settings-brave-clear-browsing-data-on-exit-page id="on-exit-tab"
```

Comment:
please make this patch one line or maybe @petemill has some ideas to reduce it

Reply:
Couple of options:
- Similar to my other comment, use a behavior "superclass" on this module, and in the `attached` lifecycle function, manipulate the DOM to create these elements (`appendChild`, `insertBefore`, etc)...
- Or, modify the template somehow during register. Possibly something like `template.content.insertBefore`. I'm not sure if that's possible, I'd have to experiment.